### PR TITLE
Passwords

### PR DESCRIPTION
**Pull request by JordanManu**

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -1,10 +1,38 @@
-const mongoose = require("mongoose");
+var mongoose = require("mongoose"),
+  // Schema = mongoose.Schema,
+  bcrypt = require("bcrypt"),
+  SALT_WORK_FACTOR = 10;
 
-const UserSchema = new mongoose.Schema({
-  email: String,
-  password: String,
+var UserSchema = new mongoose.Schema({
```
Comment (Conor-Developer) [inline models/user.js:6]: Did you make this var? Should we change this to let?

File: models/user.js [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+  email: { type: String, required: true },
+  password: { type: String, required: true },
 });
 
-const User = mongoose.model("User", UserSchema);
+UserSchema.pre("save", function (next) {
+  var user = this;
```
Comment (Conor-Developer) [inline models/user.js:12]: var? or let?

**Review by Conor-Developer — COMMENTED** [review]
Just need to check the use of var

File: models/user.js (hunk 1 above)
Comment (JordanManu) [reply models/user.js:6]: We can, I didn't make it it was already there

**Review by Conor-Developer — APPROVED** [review]
All good!

**Review by Coral-Ann — COMMENTED** [review]
Reviewed, all looks good.

**Review by Coral-Ann — APPROVED** [review]
Approved

**Pull request merged**